dev.lasheen.qr.yml

```diff
@@ -0,0 +1,37 @@
+app-id: dev.lasheen.qr
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: qr
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
```

Comment:
what does a QR code generator need network access for?
Can you leave a note here that explains why you're requesting this permission?

Reply:
The app uses the network to check for latest updates. @muelli 
Also I thought this is a standard permission.

Reply:
and what will it do with that information? It can't update itself.

Reply:
```suggestion
```

No permission is *standard*. If you don't need it don't put it.

Reply:
I need it to notify the user to update. I mentioned this to explain why I didn't add an explanation for the permission.

Reply:
Yeah, I don't think this is the best idea. This is not Mac/Windows where users need to re-download the app to get update or perform anything specific. On modern Linux distros, flatpak updates are applied automatically. It should be removed.

Reply:
Sure @barthalion